resources/translations/sv.edn

```diff
@@ -406,19 +406,19 @@
   :search {:clear-search "Töm söktermer"
            :example-searches "Exempel på sökningar"
            :learn-more "Läs mer"
-           :search "Sök"} ;;
-  :settings {:idp-email "E-postadress från identfieringstjänst" ; TODO check
-             :no-email "(ingen e-postaddress)"
+           :search "Sök"}
+  :settings {:idp-email "E-postadress från identifieringstjänst"
+             :no-email "(ingen e-postadress)"
              :notification-email "Använd en annan e-postadress för att ta emot meddelanden"
              :save "Spara"
-             :warning-about-missing-email [:<> "Du måste " [:a {:href "/settings"} "lägga till din e-postadress i dina inställningar"] ", för att kunna ta emot meddelanden om din ansökning."]}
+             :warning-about-missing-email [:<> "Du måste " [:a {:href "/settings"} "lägga till din e-postadress i dina inställningar"] ", för att kunna ta emot meddelanden om din ansökan."]}
   :table {:ascending-order "Stigande"
           :descending-order "Sjunkande"
           :all-rows-selected "Alla rader är valda"
           :no-rows-selected "Inga rader är valda"
           :some-rows-selected "Några rader är valda"
           :show-all-n-rows "Visa alla %1 rader"}
-  :unauthorized-page {:unauthorized "Ingen användningsrätt"
+  :unauthorized-page {:unauthorized "Inga användningsrättigheter"

```

Comment:
I think the plural might not be right here. This message is shown when a user tries to access a page they can't see, so it's not about entitlements granted by REMS.

Reply:
Ah, I see. Using singular for this specific word is a bit weird, though. Maybe "Ingen tillgång" would be better, or even better "Du har inte tillgång till denna sida".

Reply:
`:unauthorized` is the title of the page so "Ingen tillgång" sounds great for that. `:you-are-forbidden` could be the longer "Du har inte tillgång ..." text.

Reply:
Just pushed a commit with that :+1: